src/LibraryManager.Vsix/Json/Completion/SimpleCompletionEntry.cs

```diff
@@ -4,16 +4,13 @@
 using Microsoft.JSON.Editor.Completion;
 using Microsoft.VisualStudio.Imaging.Interop;
 using Microsoft.VisualStudio.Language.Intellisense;
-using Microsoft.Web.Editor.Completion;
 using System.Windows.Media;
 using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.Web.LibraryManager.Vsix
 {
     internal class SimpleCompletionEntry : JSONCompletionEntry
```

Comment:
>SimpleCompletionEntry [](start = 19, length = 21)

This class should remain as it was in order to let providers customize the order of completion items.

Reply:
Will do.

---
In reply to: [198666680](https://github.com/aspnet/LibraryManager/pull/258#discussion_r198666680) [](ancestors = 198666680)